src/edu/ucsb/nceas/metacat/restservice/v2/CNResourceHandler.java

```diff
@@ -566,7 +573,7 @@ private void getLog() throws InvalidToken, ServiceFailure, NotAuthorized,
         }
 
         logMetacat.debug("calling getLogRecords");
-        Log log = CNodeService.getInstance(request).getLogRecords(session,
+        org.dataone.service.types.v2.Log log = CNodeService.getInstance(request).getLogRecords(session,
```

Comment:
I added new imported class org.apache.commons.logging.Log since each ResourceHandler class has its own log object. So there are duplicate Log class and I have to use the full name here.